### ISSUE TITLE
Nova trying to import wrong Column

### ISSUE THREAD

**Issue by LorenzoSapora** [dcasia/nova-mega-filter#10]
Nova: 2.9.3
Laravel: 5.8.*

---

Adding your package to a resource

```php
<?php

namespace App\Nova;
[...]
use DigitalCreative\MegaFilter\MegaFilter;
use DigitalCreative\MegaFilter\HasMegaFilterTrait;

class User extends Resource
{
    use HasMegaFilterTrait;
[...]
    public function cards(Request $request)
    {
        return [
            MegaFilter::make([
                'filters' => [
                    (new RankFilter),
                    (new Statusfilter),
                ],
                'columns' => [
                    Column::make('Customer Name', 'user'),
                ]
            ])
        ];
    }
```

Nova throws an error saying it cannot find `App\Nova\Column`.

I feel like I'm missing something..

### Temporary fix:

Adding `use DigitalCreative\MegaFilter\Column;`

**Comment by milewski** [comment]
@LorenzoSapora looks like I forgot to add `use DigitalCreative\MegaFilter\Column;` on the main example on the documentation, but `Column` is indeed a class that ships with this class you need to import it accordingly

**Comment by LorenzoSapora** [comment]
Fixed in #13
